langchain/schema/__init__.py

```diff
@@ -1,6 +1,5 @@
 from langchain.schema.agent import AgentAction, AgentFinish
 from langchain.schema.document import BaseDocumentTransformer, Document
-from langchain.schema.language_model import BaseLanguageModel
```

Comment:
This creates a circular dependency, but as far as I can tell we've only added this here recently so no harm?

Reply:
BaseLanguageModel and callbacks I think